contracts/mainnet/connectors/morpho-blue/helpers.sol

```diff
@@ -0,0 +1,193 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "./interface.sol";
+import "../../common/stores.sol";
+import "../../common/basic.sol";
+import "../../common/interfaces.sol";
+
+abstract contract Helpers is Stores, Basic {
+	IMorpho public constant MORPHO_BLUE =
+		IMorpho(0x777777c9898D384F785Ee44Acfe945efDFf5f3E0); // TODO: Update
+
+	uint256 internal constant MARKET_PARAMS_BYTES_LENGTH = 5 * 32;
+
+	/// @dev The number of virtual assets of 1 enforces a conversion rate between shares and assets when a market is
+	/// empty.
+	uint256 internal constant VIRTUAL_ASSETS = 1;
+
+	/// @dev The number of virtual shares has been chosen low enough to prevent overflows, and high enough to ensure
+	/// high precision computations.
+	uint256 internal constant VIRTUAL_SHARES = 1e6;
+
+	enum Mode {
+		Collateral,
+		Repay,
+		Other // Neither collateral nor repay
```

Comment:
Neither collateral nor repay mode could be named `Loan`

Reply:
Even more specific as it is used only upon `supply`: `Supply`

Reply:
Agreed! [https://github.com/Instadapp/dsa-connectors-2.0/pull/1/files#diff-57afa2d25dc241f0573736f488fd0f69847c6dfeeffa95efbdd7bc0fe9fec13eR35](https://github.com/Instadapp/dsa-connectors-2.0/pull/1/files#diff-57afa2d25dc241f0573736f488fd0f69847c6dfeeffa95efbdd7bc0fe9fec13eR35)

Reply:
Resolved!